src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -198,7 +214,7 @@ full Bluetooth UUID or a short 16- or 32-bit form to `service.getCharacteristic`
 
 Writing to a Bluetooth GATT Characteristic is as easy as reading it. This time,
 let's reset the value of the Energy Expended field in the Heart Rate
-Measurement characteristic to 0 on a heart rate monitor device.
+Control Point characteristic to 0 on a heart rate monitor device.
```

Comment:
The Energy Expended field does live in the [Heart Rate Measurement](https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicViewer.aspx?u=org.bluetooth.characteristic.heart_rate_measurement.xml); it's just that you use the Control Point to reset it. You could write "let's use the Heart Rate Control Point to reset the value of the Energy Expended field to 0."